docs/libvirt-pod-networking.md

```diff
@@ -4,7 +4,7 @@ Author: Fabian Deutsch \<fabiand@redhat.com\>
 
 ## Introduction
```

Comment:
I am still missing the real need here.
Why does the VM need access to the pod network?
Why does the VM need to use the pod network IP pool? Does Kubernetes supports multiple addresses per pod, and why?